### ISSUE TITLE
Presentation peer review

### ISSUE THREAD

**Issue by osunrinde** [INFO523-Fall25-101-201/final-project-min-set-khant-solo#3]
The following is the peer review of the Presentation proposal by Min set Khant. This is a solo project
- Content: Is the research question well designed and is the data being used relevant to the research question?
- Yes, the research questions were thought of and properly designed. Also, the data utilized to solve the problem defined in this research was relevant 

- Content: Did the team use appropriate visualizations and did they interpret them accurately?
- The project owner used relevant visualizations like confusion matrix, boxplots, time-series plot and histogram plot to interpret and analyzed his data. The EDA was detailed enough. Though other relationships could be explored in the future.

- Creativity and Critical Thought: Is the project carefully thought out? Are the limitations carefully considered? Does it appear that time and effort went into the planning and implementation of the project?
- Yes, a decent amount of time was put into this project. Though, personally, I will prefer to further split the train data into another train and validation dataset and use that for the model training. Once the final model is selected, then train the final model on the entire data set. Also, I will suggest you deploy your model on a web dashboard or a simple GUI for easier interaction by the model user.

- Slides: Are the slides well organized, readable, not full of text, featuring figures with legible labels, legends, etc.?
- Yes.

- Professionalism: How well did the team present? Does the presentation appear to be well practiced? Are they reading off of a script? Did everyone get a chance to say something meaningful about the project?
- The presentation seems pretty okay to me. He explained the details of the project clearly and was not reading off a prepared script.

**Comment by min-setkhant** [comment]
> The following is the peer review of the Presentation proposal by Min set Khant. This is a solo project
> 
> * Content: Is the research question well designed and is the data being used relevant to the research question?
> * Yes, the research questions were thought of and properly designed. Also, the data utilized to solve the problem defined in this research was relevant
> * Content: Did the team use appropriate visualizations and did they interpret them accurately?
> * The project owner used relevant visualizations like confusion matrix, boxplots, time-series plot and histogram plot to interpret and analyzed his data. The EDA was detailed enough. Though other relationships could be explored in the future.
> * Creativity and Critical Thought: Is the project carefully thought out? Are the limitations carefully considered? Does it appear that time and effort went into the planning and implementation of the project?
> * Yes, a decent amount of time was put into this project. Though, personally, I will prefer to further split the train data into another train and validation dataset and use that for the model training. Once the final model is selected, then train the final model on the entire data set. Also, I will suggest you deploy your model on a web dashboard or a simple GUI for easier interaction by the model user.
> * Slides: Are the slides well organized, readable, not full of text, featuring figures with legible labels, legends, etc.?
> * Yes.
> * Professionalism: How well did the team present? Does the presentation appear to be well practiced? Are they reading off of a script? Did everyone get a chance to say something meaningful about the project?
> * The presentation seems pretty okay to me. He explained the details of the project clearly and was not reading off a prepared script.

**Comment by min-setkhant** [comment]
Thank you very much for your detailed peer review and feedback! I really appreciate your time in evaluating my presentation and proposal.

I’m glad to hear that the research questions, data, and visualizations were relevant and well-presented. I also added a Price Predictor to make the project more interactive, which allows users to input features and get predicted BMW prices, this was part of implementing your suggestion for a more usable interface.

I also appreciate your suggestions regarding splitting the training data into train/validation sets and deploying the model on a web dashboard or GUI , these are great ideas for improving the project further. I’ll definitely consider exploring additional relationships in the EDA and implementing your recommendations where possible.

Thanks again for your constructive feedback!
